src/main/java/net/glowstone/block/ItemTable.java

```diff
@@ -234,7 +234,6 @@ private void registerBuiltins() {
         reg(Material.BREWING_STAND_ITEM, new ItemPlaceAs(Material.BREWING_STAND));
         reg(Material.CAULDRON_ITEM, new ItemPlaceAs(Material.CAULDRON));
         reg(Material.FLOWER_POT_ITEM, new ItemPlaceAs(Material.FLOWER_POT));
-        reg(Material.SKULL_ITEM, new ItemPlaceAs(Material.SKULL));
```

Comment:
A diff is your changes from the previous revision of a file. I am just saying that it isn't necessary to move the line down.